etc/temp-ldap-setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Why is this file called temp-ldap-setup.sh? Why not setup-ldap.sh?

Reply:
Didn't expect to merge, named temp because would've been deleted